### ISSUE TITLE
Clarify instructions for step 81 of spreadsheet project

### ISSUE THREAD

**Issue by jdwilkin4** [freeCodeCamp/freeCodeCamp#53224]
### Describe the Issue

We received this comment on the [forum](https://forum.freecodecamp.org/t/feedback-learn-functional-programming-by-building-a-spreadsheet/663204) concerning the directions for step 81 of the spreadsheet project

>Improve message in check code dialog.

>The code:
```js
const toNumberList = args => args.split(',').map(el => parseFloat(el));
```
>is erroring because .map() doesn’t need a callback parameter but just a reference to callback and I think that is a good opportunity to reinforce an earlier point.

We should update the instructions to better clarify what should be passed into the `map` method

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-functional-programming-by-building-a-spreadsheet/step-81

### Your code

see explanation above

### Expected behavior

see explanation above

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

**Comment by Njong392** [comment]
I have to agree with the miswording because it took me a small while to pass the challenge. But at some point, the hint says 

> You should pass a reference to parseFloat as the callback to .map().

So what if we specify in the instruction, that `parseFloat` should be passed as a reference instead? Maybe something like,

> Declare a toNumberList function that takes an args parameter and returns the result of splitting the args by commas. Map the resulting array to parseFloat by passing it as a reference.

**Comment by jdwilkin4** [comment]
> Declare a toNumberList function that takes an args parameter and returns the result of splitting the args by commas. Map the resulting array to parseFloat by passing it as a reference.

yeah, I think that is better. 👍 
I think what's missing too is that the test is expecting an implicit return and chaining the `split` and `map` methods.

If someone were to write this, then they wouldn't be able to pass

```js
const toNumberList = args => {
const splitArr = args.split(',')
return splitArr.map(parseFloat)
}
```
This updated description could work to better indicate that the tests are looking for a specific solution

```md
Declare a `toNumberList` function that takes an `args` parameter and implicitly returns the result of splitting the args by commas. Then chain the `map` method to your `split` method and pass in `parseFloat` as a reference.

**Comment by Njong392** [comment]
Sounds good. Is this open for contribution yet? Or we wait for any other contributions?

**Comment by jdwilkin4** [comment]
I just opened it.
If we need to make changes, then we just iterate from there on the PR itself 👍
